srcpkgs/walk/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'walk'
+pkgname=walk
+version=20190920
+revision=1
+_commit=70f7a8c104acf109ae810a9b34eb8e8b92b4d27d
+build_style="gnu-makefile"
+wrksrc="${pkgname}-${_commit}"
+short_desc="walk and sor utility, collectively replacing find"
+maintainer="orphaned"
+license="Apache-2.0"
+homepage="https://github.com/google/walk"
+distfiles="https://github.com/google/walk/archive/${_commit}.tar.gz"
+checksum=fce16683d757605a11284586794627a4352ec62718c59093a02f56b39896a1cd
+
+do_build() {
+    make
```

Comment:
Is there an issue with the `do_build` provided by `build_style=gnu-makefile`

Reply:
Only that I did not know about it.